src/blocks/server.blocks/middleware/__page-title/middleware__page-title.node.js

```diff
@@ -2,30 +2,40 @@ modules.define('middleware__page-title', ['config', 'logger', 'model'], function
 
     logger = logger(module);
 
-    provide(function () {
-        return function (req, res, next) {
-            logger.debug('get title by request %s', req.url);
+    provide({
+        run: function () {
```

Comment:
1. Templates takes title of the page from `req.__data.title`. The middleware of forum where we can set title is early in list, so in this case, if `req.url` match forum, we skip setting default title, because we did it early. 
2. Middleware called `page-title` and did one function - setting title of the page. 
`Run` is default method for every middleware that contain in our case main things, another method `hasExceptions ` created for create opportunities rewrite this code on another level (bem-info) to set specific title.